src/features/GuestAccess/GuestAccess.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  Container,
+  Stack,
+  InputLabel,
+  TextField,
+  Button,
+  Typography,
+  Select,
+  MenuItem,
+  FormControl,
+  Paper,
+  Box,
+} from "@mui/material";
+import { Link } from "react-router";
+import VpnKeyIcon from '@mui/icons-material/VpnKey';
+
+const GuestAccess = () => {
+  return (
+    <Container
+      sx={{
+        height: "600px",
+        marginTop: "60px",
+      }}
+    >
+      <Paper
+        sx={{
+          padding: "20px",
+          borderRadius: "10px",
+        }}
+      >
+        <Stack
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+            width: { xs: "auto", sm: 500 },
+            height: "100%",
+            margin: "0 auto",
+            gap: "30px",
+          }}
+        >
+
+          <Box 
+            sx={{ 
+              display: "flex", 
+              justifyContent: "center",
+              marginBottom: "-20px"
+            }}
+          >
+            <VpnKeyIcon 
+              sx={{ 
+                fontSize: 60, 
+                color: "primary.main",
+              }} 
+            />
+          </Box>
+          
+          <Typography
+            sx={{
+              textAlign: "center",
+              fontWeight: 500,
+              fontSize: "1.5rem",
+              lineHeight: "1.2",
+            }}
+          >
+            Guest Access
+          </Typography>
+          
+          <Typography 
+            variant="body2" 
+            color="text.secondary"
+            sx={{ 
+              textAlign: "center",
+              marginTop: "-20px",
+              marginBottom: "10px"
+            }}
+          >
+            Generate a temporary access key for your visitors
+          </Typography>
+          
+          <Stack
+            sx={{
+              width: "90%",
+              gap: "20px",
+            }}
+          >
+            <Stack
+              sx={{
+                width: "100%",
+                gap: "10px",
+              }}
+            >
+              <InputLabel>Guest Name</InputLabel>
+              <TextField
+                sx={{
+                  width: "90%",
```

Comment:
nitpick: can the width be 100% so the text aligns left of the form? Or if the TextField is intended to be 90%, update the InputLabel to 90% to match